querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -686,29 +588,76 @@ export const QueryEditor: React.FC<
         );
         /* ---- end of <ReactCodeMirror /> properties ---- */
 
-        const editorClassName = clsx({
-            fullScreen,
-            [className]: !!className,
-        });
+        const renderLintButtons = () => {
+            if (isLinting) {
+                return (
+                    <span className="flex-row mr8">
+                        <Icon name="Loading" className="mr4" size={16} />
+                        Linting
```

Comment:
do user really care about when it's linting? also as it will happen so frequently, it will be a bit distracting, so I think it's just enough to show the number of errors when the lint finishes.

Reply:
Do you think just a loading icon would be good enough? The reason why I want to put it is because some user complained that they don't know if the linter is actually running, also we do plan to add linters that will take more time to run compare to the existing ones

Reply:
okay, then let's keep it as is for now and see how it will look like actually. with only a loading icon, people may not know what is loading.